docs/access-handler/providers.md

```diff
@@ -0,0 +1,164 @@
+## Providers
+
+The access handler uses a provider framework which allows new services to be integrated easily by implementing a specific interface.
+
+### Adding a provider
+
+Providers are defined in accesshandler/pkg/providers if this provider is for a service which may have multiple possible providers, create a parent folder and
+then create a specific folder for your new provider. E.g aws/sso or azure/ad otherwise, a single folder is ok e.g okta
```

Comment:
```suggestion
Providers are defined in accesshandler/pkg/providers if this provider is for a service which may have multiple possible providers, create a parent folder and then create a specific folder for your new provider. E.g aws/sso or azure/ad otherwise, a single folder is ok e.g okta
```